docs/celldive/index.md

```diff
@@ -28,7 +28,7 @@ The other fields function the same way for all assays using antibodies. For more
 
 
 
-In the portal: [Cell DIVE](https://portal.hubmapconsortium.org/search?mapped_data_types%5B0%5D=Cell+DIVE&entity_type%5B0%5D=Dataset)
+In the portal: [Cell DIVE](https://portal.hubmapconsortium.org/search?mapped_data_types%5B0%5D=Cell+DIVE&entity_type%5B0%5D=Dataset) / [Cell DIVE](https://portal.hubmapconsortium.org/search?mapped_data_types%5B0%5D=Cell+DIVE&entity_type%5B0%5D=Dataset)
```

Comment:
Should what is in the added brackets be [cell-dive]?

Reply:
This is dynamically created from the yaml files. So, no